src/services/UserService.ts

```
@@ -0,0 +1,23 @@
+// services/UserService.ts
+export class UserService {
+    private static readonly BASE_URL = "http://localhost:8080";
+  
+    static async getUserProfile(token: string) {
+      const res = await fetch(`${this.BASE_URL}/api/users/me`, {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)